balancer-js/src/modules/pools/queries/queries.integration.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { expect } from 'chai';
+import { BalancerSDK, Network, PoolType } from '@/.';
+import { AddressZero, MaxUint256, Zero } from '@ethersproject/constants';
+import { bn } from '@/lib/utils';
+import { ParamsBuilder } from '.';
+
+const rpcUrl = 'http://127.0.0.1:8545';
+const network = Network.MAINNET;
+const sdk = new BalancerSDK({ network, rpcUrl });
+const { contracts } = sdk;
+
+const stETHPool = {
```

Comment:
Would be good to add a ComposableStable test case. There were some issues when these were originally launched as the request needed a specific format for phantom token. These were my notes:

```
JoinPoolRequest:
assets: Needs to have phantomBpt token in array (in sorted place)
maxAmountsIn: Needs a value for phantomBpt indice
UserData:
amountsIn: Should not have a value for phantomBpt token
```

Reply:
good point, will add test cases for that.

Reply:
test case added and all works.

one thing: for now we don't execute contract calls. that means that inputs should be changed by the params builder, because outputs will change as well and we wouldn't be able to know about it. that's why tokenList sorting needs to happen before passing it to the builder.

Reply:
Just to clarify - token amounts arrays should have correct ordering? Can we add this to docs for the `ParamsBuilder` functions?

Reply:
done